cmd/landscaper-controller/app/app.go

```diff
@@ -93,6 +95,20 @@ func (o *Options) run(ctx context.Context) error {
 		opts.MetricsBindAddress = fmt.Sprintf(":%d", o.Config.Metrics.Port)
 	}
 
+	if o.Config.DeployerManagement.DeployerRepositoryContext == nil {
+		if o.Config.Controllers.Contexts.Config.Default.RepositoryContext != nil {
+			o.Config.DeployerManagement.DeployerRepositoryContext = o.Config.Controllers.Contexts.Config.Default.RepositoryContext.DeepCopy()
+		} else if o.Config.RepositoryContext != nil {
+			o.Config.DeployerManagement.DeployerRepositoryContext = o.Config.RepositoryContext.DeepCopy()
+		} else {
+			defaultCtx, err := cdv2.NewUnstructured(cdv2.NewOCIRegistryRepository("eu.gcr.io/gardener-project/development", ""))
```

Comment:
we should make that default in the configuration defaulting here: https://github.com/gardener/landscaper/blob/master/apis/config/v1alpha1/defaults.go#L63

Reply:
I have moved the coding